src/core/tests/preprocess.cpp

```diff
@@ -1170,6 +1170,29 @@ TEST(pre_post_process, postprocess_convert_layout_invalid_dims_dyn_shape) {
 
 // Postprocessing - other
 
+TEST(pre_post_process, postprocess_preserve_rt_info) {
```

Comment:
what is the behavior for parameters? is it covered with tests?

Reply:
Yes, there were fixed by https://github.com/openvinotoolkit/openvino/pull/8721 , and the same considerations for results were missed